AppHub.podspec

```diff
@@ -0,0 +1,46 @@
+#
+# Be sure to run `pod lib lint AppHub.podspec' to ensure this is a
+# valid spec before submitting.
+#
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = "AppHub"
+  s.version          = "0.2.2"
+  s.summary          = "A short description of AppHub."
+
+# This description is used to generate tags and improve search results.
+#   * Think: What does it do? Why did you write it? What is the focus?
+#   * Try to keep it short, snappy and to the point.
+#   * Write the description between the DESC delimiters below.
+#   * Finally, don't worry about the indent, CocoaPods strips it!
+  s.description      = <<-DESC
```

Comment:
Please add this as the description:

"AppHub lets you instantly update React Native apps without resubmitting to the App Store."